src/VsCodeDialog/vscode.ts

```diff
@@ -80,7 +90,18 @@ export const getVsCodeStatus = () => async (dispatch: Dispatch) => {
             status = VsCodeStatus.MISSING_EXTENSIONS;
         else {
             const nrfjprog = await getNrfjprogStatus();
-            if (!nrfjprog) status = VsCodeStatus.MISSING_NRFJPROG;
+            const vscode = await spawnAsync(
+                'file "$(dirname "$(readlink $(which code))")/../../../MacOS/Electron"'
```

Comment:
Have you tested this on windows? I don't think `file` is part of the standard commands.

Reply:
Seems I forgot about that part. I'll just move it into the if isAppleSilicon below.